src/toffy/fov_watcher.py

```diff
@@ -49,18 +52,16 @@ def __init__(self, run_folder: str, timeout: int = 10 * 60):
             if run_order * scan < 0:
                 raise KeyError(f"Could not locate keys in {run_folder}.json")
 
-            fov_names = [f"fov-{run_order}-scan-{s + 1}" for s in range(scan)]
-
-            # identify moly points
+            # scan 2's don't contain significant imaging data per new MIBI specs
+            fov_name = f"fov-{run_order}-scan-1"
             if fov.get("standardTarget", "") == "Molybdenum Foil":
-                for fov_name in fov_names:
-                    self.moly_points.append(fov_name)
+                self.moly_points.append(fov_name)
+
+            self.fov_progress[fov_name] = {"json": False, "bin": False}
 
-            for fov_name in fov_names:
-                self.fov_progress[fov_name] = {
-                    "json": False,
-                    "bin": False,
-                }
+        # compute the highest FOV number, needed for checking if final FOV processed
+        fov_nums = sorted([int(f.split("-")[1]) for f in self.fov_progress.keys()])
+        self.highest_fov = fov_nums[-1]
```

Comment:
Since we are only adding scan 1's to `fov_progress`, can we simply do `self.highest_fov = len(self.fov_progress.keys())`?